docs/source/configuration/authorization.mdx

```diff
@@ -1,78 +1,209 @@
 ---
 title: Authorization in the Apollo Router
+description: Strengthen your supergraph's security with advanced access controls
+minVersion: 1.27.0
 ---
 
-> ⚠️ **This is an [Enterprise feature](../enterprise-features/) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).
->
-> If your organization _doesn't_ currently have an Enterprise plan, you can test out this functionality by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
+<GraphOSEnterpriseRequired />
 
-The Apollo Router supports graph based authorization policies, through the `@authenticated` and `@requiresScopes` directives. They are used to authorize access to specific types or fields.
+<FeatureInPreview />
 
-They are defined as follows:
+APIs provide access to business-critical data. Unrestricted access can result in data breaches, monetary losses, or potential denial of service. Even for internal APIs, checks can be essential to limit data to authorized parties.
 
-```graphql
-directive @authenticated on OBJECT | FIELD_DEFINITION | INTERFACE | SCALAR | ENUM
+Enforcing authorization in the Apollo Router is valuable for a few reasons:
+- Validating authorization _before_ processing requests allows for early request termination.
+- It supports a defense-in-depth strategy by creating an initial checkpoint that other service layers can reinforce.
+
+The Apollo Router provides fine-grained access controls at your graph's edge. Using the `@authenticated`, `@requiresScopes`, and `@policy` directives, you can define access to specific fields and types across your supergraph:
+
+- The [`@authenticated`](#authenticated) directive works in a binary way: authenticated requests can access the specified field or type, and unauthenticated requests can't.
+- The [`@requiresScopes`](#requiresscopes) directive allows granular access control through the scopes you define.
+- The [`@policy`](#policy) directive offloads authorization validation to a [Rhai script](../customizations/rhai/) or a [coprocessor](../customizations/coprocessor) and integrates the result in the router. It's useful when your authorization policies go beyond simple authentication and scopes. 
+
+You define and use these directives on subgraph schemas, and GraphOS [composes](#composition-and-federation) them onto the supergraph schema.
+The router then enforces these directives on all incoming requests.
+
+## Prerequisites
```

Comment:
Can we simply the prerequisites section? List the two simple options

- JWT Authentication configuration
- Claims extraction and augmentation via coprocessors

And link them to their description sections elsewhere. Right now I feel like I need to read through a lot before I can get to the exciting bit

Reply:
Hmm, I'm not sure putting them on a different page or at the end of the article would make a lot of sense.

We can put them in Expansion Blocks so they're "hidden" and users have to click to expand. Let me know if you think that's insufficient.